src/main/java/edu/indiana/dlib/amppd/model/Content.java

```diff
@@ -17,6 +19,7 @@
  */
 @MappedSuperclass
 @Index(members={"externalSource","externalId"}, unique="true")
+@UniqueExternalSrcAndId(message="External Source and ID must be unique within its parent collection")
 @Data
 @EqualsAndHashCode(callSuper=true)
 @ToString(callSuper=true)
```

Comment:
We don't need to add constraints in Content. The UniqueName and @UniqueExternalSrcAndId are added in each subclass, since they are different per entity class. For now, we are adding @UniqueExternalSrcAndId to only Item, not Collection, since the latter doesn't have the fields populated or used (pending on functional decisions later).

Reply:
fixed